homeassistant/components/mqtt_room/sensor.py

```diff
@@ -28,20 +29,25 @@
 
 ATTR_DISTANCE = "distance"
 ATTR_ROOM = "room"
+ATTR_UPDATED = "updated"
+ATTR_EXTRA_ROOMS = "extra_rooms"
 
 CONF_AWAY_TIMEOUT = "away_timeout"
+CONF_EXTRA_ROOMS = "extra_rooms"
 
 DEFAULT_AWAY_TIMEOUT = 0
 DEFAULT_NAME = "Room Sensor"
 DEFAULT_TIMEOUT = 5
 DEFAULT_TOPIC = "room_presence"
+DEFAULT_EXTRA_ROOMS = 0
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_DEVICE_ID): cv.string,
         vol.Required(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
         vol.Optional(CONF_AWAY_TIMEOUT, default=DEFAULT_AWAY_TIMEOUT): cv.positive_int,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_EXTRA_ROOMS, default=DEFAULT_EXTRA_ROOMS): cv.positive_int,
```

Comment:
@jbouwh CONF_EXTRA_ROOMS is (ultimately) being used on line 116.

The reason I decided to make this configurable is primarily the "Danger" box in this section of the developer documentation: https://developers.home-assistant.io/docs/core/entity/#generic-properties

Basically I want to avoid adding additional extra_attributes for users who have no intention of actually using them.